tests/test_parsley.py

```
@@ -1,97 +1,399 @@
+        assert msg_data == b'\x00\x9a\xbc'
+        
+    def test_parse_bitstring_empty(self):
+        bit_str = BitString() #just an empty bitstring
+        with pytest.raises(IndexError) as e:
+            parsley.parse_bitstring(bit_str)
+        # message content is not asserted as an index error is expected
+        
+    def test_parse_bitstring_small(self):
+        bit_str = BitString()
+        bit_str.push(b'\xFF', 8) # only 8 bits, less than required 29 bits for SID
+        with pytest.raises(IndexError) as e:
```

Ruff (0.14.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)